### DISCUSSION TITLE
Optimize font/css/icon file size

### DISCUSSION THREAD

**Discussion by SKFrozenCloud** [quasarframework/quasar#11796] [Ideas / Proposals] [answered]
If you use an icon, Quasar will include the **whole** file which is big.

Is there a way to include the icons/css you use in your app?

**Comment by rstoenescu** [comment, marked as answer]
It matters what you choose to use. Webfonts are supplied through a webfont file (so all icons, regardless of your usage). On the other hand, if you use the svg form (check `@quasar/extras` in svg form) you'll get only the icons that you actually use, but the downside is that you need to manually import the svg icons everywhere you use them.

https://quasar.dev/vue-components/icon#svg-icons

**Reply by SKFrozenCloud** [reply]
The downsides could be fixed by creating a single js file that imports all the icons needed and then you could use it anywhere in your app.

**Reply by hawkeye64** [reply]
You can easily do that with a composable
